playground/frontend/lib/pages/standalone_playground/widgets/playground_page_footer.dart

```diff
@@ -57,7 +57,7 @@ class PlaygroundPageFooter extends StatelessWidget {
                 onPressed: () {
                   launchUrl(Uri.parse(kReportIssueLink));
                   PlaygroundComponents.analyticsService.sendUnawaited(
-                    ReportIssueAnalyticsEvent(
+                    IssueReportedAnalyticsEvent(
```

Comment:
```suggestion
                    ReportIssueClickedAnalyticsEvent(
```
It is not reported yet.